src/pyconverter/xml2py/cli.py

```diff
@@ -24,13 +24,13 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
     target_path: str, optional
         Path to the directory where you want the autogenerated package to be created.
```

Comment:
```suggestion
        Path to the directory where you want to create the autogenerated package.
```
Avoid future tense where possible.